src/main/java/dansplugins/simpleskills/commands/SkillCommand.java

```diff
@@ -1,40 +1,42 @@
 package dansplugins.simpleskills.commands;
 
+import dansplugins.simpleskills.AbstractSkill;
 import dansplugins.simpleskills.data.PersistentData;
-import dansplugins.simpleskills.objects.abs.Skill;
 import dansplugins.simpleskills.services.LocalMessageService;
 import org.bukkit.command.CommandSender;
 import preponderous.ponder.minecraft.abs.AbstractPluginCommand;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.Collections;
 
 /**
  * @author Daniel Stephenson
  */
 public class SkillCommand extends AbstractPluginCommand {
 
     public SkillCommand() {
-        super(new ArrayList<>(Arrays.asList("skill")), new ArrayList<>(Arrays.asList("ss.skill")));
+        super(
+                new ArrayList<>(Collections.singletonList("skill")),
+                new ArrayList<>(Collections.singletonList("ss.skill"))
+        );
     }
 
     @Override
     public boolean execute(CommandSender commandSender) {
         commandSender.sendMessage(LocalMessageService.getInstance().convert(LocalMessageService.getInstance().getlang().getString("SkillHUsage")));
-
         return false;
     }
 
     @Override
     public boolean execute(CommandSender commandSender, String[] args) {
         String skillName = args[0];
-        Skill skill = PersistentData.getInstance().getSkill(skillName);
+        AbstractSkill skill = PersistentData.getInstance().getSkill(skillName);
         if (skill == null) {
-            commandSender.sendMessage(LocalMessageService.getInstance().convert(LocalMessageService.getInstance().getlang().getString("SkillNotFound")));
-
+            commandSender.sendMessage(LocalMessageService.getInstance()
+                    .convert(LocalMessageService.getInstance().getlang().getString("SkillNotFound")));
         }
-        skill.sendInfo(commandSender);
+        // TODO fix skill.sendInfo(commandSender);
```

Comment:
What needs to be fixed here?

Reply:
Ah, okay, I'll commit a new change for that fix now :) thank you very much for finding one of the last "TODO"s I had left for myself :)